apps/vsce/src/components/engineService.ts

```diff
@@ -230,40 +226,40 @@ export class EngineService {
 			this.#onExecuteCodemodSetMessage(message);
 		});
 
-		this.__pollCodemodEngineNode();
+		this.__onCodemodEngineNodeLocated();
 	}
 
-	private async __pollCodemodEngineNode() {
-		let iterations = 0;
+	// private async __pollCodemodEngineNode() {
```

Comment:
is this commented intentionally?

Reply:
yup, because we don't require users to have a globally installed codemod library anymore